walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
       <h1>Stanley 🐈</h1>
       <p>This is Stanley.</p>
       <img src="images/stanley-1.jpg"/>
-
+    
```

Comment:
nit unnecessary space

Reply:
hahah This was actually the Google walkthrough code! :')

Reply:
Yea, so i don't expect any change to this file.